index.js

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env node
+require("dotenv").config();
+const axios = require("axios");
+
+// Usage from the command line:
+// npm install
+// node index sourceChannel targetChannel
+// e.g. node index general dev-chatter
+
+// Note: Do not include the hashtag in the channel name. Though I did account for stripping the hashtag away, process.argv returns undefined if the hashtag is present and I'm unsure of why.
+
+// Store Slack access token in a .env file in the root folder
+// You can ping a JDSB admin on Slack for the OAuth token, or request to become a collaborator on the app.
+// The token for an app can be found by selecting the app from api.slack.com/apps and navigating to "OAuth & Permissions" page after it has been installed to a workspace.
+const accessToken = process.env.SLACK_ACCESS_TOKEN;
+
+async function getConversationId(conversation) {
+
+  // Trim the pound sign if it was provided in the conversation argument
+  if (conversation.startsWith('#')) {
+    conversation = conversation.slice(1)
+  }
+
+  // Destructuring to pull out the channels result from the response
+  const { data: { channels } } = await axios.get(`https://slack.com/api/conversations.list?token=${accessToken}&pretty=1`)
+
+  // Find the channel with matching conversation name and pull out its ID
+  const conversationId = channels.find(chan => chan.name === conversation).id
+
+  return conversationId
+}
+
+async function getConversationMembers(conversationId) {
```

Comment:
See above comment. This should be getChannelMembers() now.

Reply:
Same reasoning -- Conversation API/private convos, but can do.

Reply:
We'll make improvements for the Slack App migration.